src/pages/MainPage.tsx

```diff
@@ -1,7 +1,59 @@
 import React from "react";
+import "@/styles/MainPage.scss";
+import { Spinner } from "react-bootstrap";
+import { useMainPageFetch } from "@/hooks/useCatalogFetch";
+import MainItem from "@/components/MainItem";
+import { useNavigate } from "react-router-dom";
 
 const MainPage = () => {
-  return <div>Page Main</div>;
+  const navigate = useNavigate();
+  const { data, error, loading } = useMainPageFetch();
+
+  const goToSeriesPage = (id: string) => {
+    navigate("/series/" + id);
+  };
+
+  return (
+    <React.Fragment>
+      {error ? (
+        <div className="errorDiv">Упс... Что-то пошло не так</div>
+      ) : loading ? (
```

Comment:
Nested ternary operations is Evil 
```suggestion
      {error && !loading (
        <div className="errorDiv">Упс... Что-то пошло не так</div>
      ) 
```